test/pipeline_test_runner.cc

```diff
@@ -731,18 +733,25 @@ TEST_CASE("PerPhaseTest") { // NOLINT
 
         handler.addObserved(*gs, "rbs-rewrite-tree", [&]() { return nodes->toString(*gs); });
 
-        ast::ParsedFile file;
-        switch (parser) {
-            case realmain::options::Parser::ORIGINAL: {
-                file = testSerialize(*gs, ast::ParsedFile{ast::desugar::node2Tree(ctx, move(nodes)), f.file});
-                break;
-            }
-            case realmain::options::Parser::PRISM: {
-                file = testSerialize(*gs, ast::ParsedFile{ast::prismDesugar::node2Tree(ctx, move(nodes)), f.file});
-                break;
+        auto legacyAST = ast::desugar::node2Tree(ctx, move(nodes));
```

Comment:
Could we call this `originalAST` so that it matches the `original` `--parser` option?

Reply:
Hmmm, that's not quite right, either.

The parsing here could be done by either the original parser, or Prism. "Legacy" isn't quite right, but what I'm getting at is that it uses the "normal" `Desugar.cc` logic, instead of direct translation within Prism's Translator.

Reply:
We don't really have to give this a name actually, we can just assign it right to the `ast` local.